app/javascript/components/editor/TestRunSummary.tsx

```diff
@@ -121,8 +101,78 @@ export function TestRunSummary({
 
   return (
     <>
-      <TestRunSummaryHeader testRun={testRun} />
-      <TestRunSummaryContent testRun={testRun} onCancel={cancel} />
+      <TestRunSummary.Header testRun={testRun} />
+      <TestRunSummary.Content testRun={testRun} onCancel={cancel} />
     </>
   )
 }
+
+TestRunSummary.Header = ({ testRun }: { testRun: TestRun }) => {
+  switch (testRun.status) {
+    case TestRunStatus.FAIL:
+      return (
+        <div className="summary-status failed">
+          <span className="--dot" />1 test failure
+        </div>
+      )
+    case TestRunStatus.PASS:
+      return (
+        <div className="summary-status passed">
+          <span className="--dot" />
+          All tests passed
+        </div>
+      )
+    default:
+      return null
+  }
+}
+
+TestRunSummary.Content = ({
```

Comment:
This is going to "live" update without user interaction, right?

Consider wrapping the status message in a [status role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role). What gives? It will allow screenreaders to _read out_ the change automatically when it happens, without you needing to do anything for that.

Reply:
Thanks. I've added this :)